cmd/download.go

```diff
@@ -28,7 +28,7 @@ latest solution.
 
 Download other people's solutions by providing the UUID.
 `,
-	Args: cobra.ExactArgs(1),
+	Args: cobra.ExactArgs(0),
```

Comment:
@strickinato thanks for contribution and welcome. @kytrinyx will give a through review of the changes, but I will leave a note on this line. We should get rid of the call to ExactArgs as it adds its own error messaging which is not helpful to the user. Maybe showing the default help message? What do you think your make sense?

Reply:
I think we should show a specific error message rather than the default help if you've called the command with no arguments.